src/main/resources/io/jenkins/plugins/designlibrary/sample.js

```diff
@@ -17,34 +12,42 @@ document.addEventListener("DOMContentLoaded", () => {
       fetch(fullUrl)
         .then(response => response.text())
         .then(text => {
-          const language = extractLanguageFromClassList(element);
-          if (language.length > 0) {
-            element.innerHTML = Prism.highlight(text, Prism.languages[language], language.pop())
-          } else {
-            element.innerHTML = text
+          element.innerText = text
+
+          Prism.highlightElement(element)
+
+          function setPrismBackgroundVariable() {
+            const computedStyle = window.getComputedStyle(element.parentElement)
+            const background = computedStyle.getPropertyValue('background')
+
+            document.documentElement.style
+              .setProperty(prismVariable, background);
+          }
+
+          const prismVariable = '--prism-background'
+          if (!getComputedStyle(document.documentElement).getPropertyValue(prismVariable)) {
+            setPrismBackgroundVariable()
+
+            if (window.isSystemRespectingTheme) {
+              window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', event => {
+                setPrismBackgroundVariable()
+              });
+            }
```

Comment:
Wouldn't it be sufficient to call `Prism.highlightElement(element)` when the user changes the theme?

Reply:
No I've added some comments to explain more but the above isn't for the elements it's so that the copy to clipboard section matches which doesn't use prism:
![image](https://github.com/jenkinsci/design-library-plugin/assets/21194782/6641608b-8974-4f8d-a815-42cb909d2cea)